vnext/ReactUWP/Views/FlyoutViewManager.cpp

```diff
@@ -108,6 +108,7 @@ class FlyoutShadowNode : public ShadowNodeBase {
 
   winrt::Flyout::Closing_revoker m_flyoutClosingRevoker{};
   winrt::Flyout::Closed_revoker m_flyoutClosedRevoker{};
+  int64_t m_token;
```

Comment:
>m_token [](start = 10, length = 7)

Can you rename this to something more descriptive, like m_tokenContentPropertyChangeCallback?

Reply:
Done